### ISSUE TITLE
[feature request] Highlight a word then search the higlighted word in th current webpage

### ISSUE THREAD

**Issue by h3ndry** [gdh1995/vimium-c#361]
Is it possible to implement the functionality similar to * # in vim normal mode?
basically in vim, you can search the word under the cursor by pressing * or # in normal mode

since a browser is different from vim, I thought how about we implement the ability to search a highlight word by pressing */# in visual mode

**Comment by gdh1995** [comment]
If you have selected a word, then "/" should enter FindMode and use it as the initial query.



---Original---
From: "Hendry ***@***.***&gt;
Date: Fri, Jun 4, 2021 20:07 PM
To: ***@***.***&gt;;
Cc: ***@***.***&gt;;
Subject: [gdh1995/vimium-c] [feature request] Hilight a word then search the higlight word in th current webpage (#361)




 
Is it possible to implement the functionality similar to * # in vim normal mode?
 basically in vim, you can search the word under the cursor by pressing * or # in normal mode
 
since a browser is different from vim, I thought how about we implement the ability to search a highlight word by pressing */# in visual mode

**Comment by gdh1995** [comment]
When in FindMode, you may use `Ctrl+Shift+J` / `Ctrl+Shift+K` to highlight all visible matches (for a while).

**Comment by h3ndry** [comment]
I just tried it, it doesn't work in my case. I use Firefox

**Comment by gdh1995** [comment]
The effect of `Ctrl+Shift+J` will look like:

![image](https://user-images.githubusercontent.com/5547703/121457018-90d69400-c9d9-11eb-8903-438fba63aa94.png)

What's your system name? I'm using Win 10 and testing Firefox 88.

**Comment by h3ndry** [comment]
To me doesn't work, when i press `ctr+shift+j` it open consol log
![image](https://user-images.githubusercontent.com/51496873/121801571-814c8a80-cc27-11eb-8a9c-2af38ee3312a.png)

I am using Arch Linux and Firefox 89

**Comment by gdh1995** [comment]
what about ctrl-shift-k ?
if you use "mapkey <f6:f&gt; <c-s-j&gt;", then f6 will be treated by Vimium C as ctrl-shift-j in find mode.



---Original---
From: "Hendry ***@***.***&gt;
Date: Sun, Jun 13, 2021 17:14 PM
To: ***@***.***&gt;;
Cc: "Dahan ***@***.******@***.***&gt;;
Subject: Re: [gdh1995/vimium-c] [feature request] Highlight a word then search the higlighted word in th current webpage (#361)




 
To me doesn't work, when i press ctr+shift+j it open consol log
 
 
I am using Arch Linux and Firefox 89
